app/src/main/java/org/breezyweather/sources/gadgetbridge/GadgetbridgeService.kt

```diff
@@ -122,6 +122,14 @@ class GadgetbridgeService @Inject constructor() : BroadcastSource {
                 maxTemp = day.day?.temperature?.temperature?.roundCelsiusToKelvin(),
                 minTemp = day.night?.temperature?.temperature?.roundCelsiusToKelvin(),
 
+                // TODO: humidity
```

Comment:
What's a daily humidity? The maximum humidity of that day (humid)? The minimum (dryest)? The median? The average? (so if I have 80% of humidity the morning and 20% on the evening, the average is 50%?)

Reply:
I am honestly not sure. I see that APIs such as openweathermap have a single "humidity" value for each day, but I am not aware of which of those it refers to, and could not find it documented anywhere. Do you know?

Reply:
I have no idea. I received a request to implement it previously, but no one was able to give me a clear definition of a "daily humidity". I don't find any of the options (max, min, average) good neither, that's why I never implemented it. Feel free to do your own computing based on hourly data in the `GadgetbridgeService` itself though

Reply:
I don't think it makse sense to compute a measurement that we can't reach an agreement about what makes sense. I added the TODO mostly to indicate that there's currently a field in the weatherspec that is not mapped - but as far as I know no gadgets display it for now.

Reply:
I mean if you end up having a clear definition on what you expect weather apps to provide on Gadgetbridge side, but we don't or we have a different definition, you can implement your own definition ;)

Reply:
Looking at the values returned by the APIs used by the gadgets, they seem to match the "daily humidity" returned by websites. As to what that means, I don't know :)

Will revisit if we ever actually use the value.